modules/80_optimization/testOneRegi/equations.gms

```diff
@@ -13,14 +13,17 @@ q80_budg_intertemp(regi)..
 0 =e=  
  SUM(ttot$(ttot.val ge 2005), 
     pm_ts(ttot) 
-       * SUM(trade$(NOT tradeSe(trade)), 
+       * (
+         SUM(trade$(NOT tradeSe(trade)), 
```

Comment:
why do you use here tradeSe and not tradeCap?

Reply:
First, I didn't change this, this specific line is obviously just adding a bracket. The actual change is further below, where I'm adding new terms.

Second, as explained above, Renato chose to add `tradeSe` to `trade` so he had to exclude it at that point. `tradeCap` doesn't need to be included at this point, there should be further terms below this line.